sonar-dotnet-shared-library/src/test/java/org/sonar/plugins/dotnet/tests/WildcardPatternFileProviderTest.java

```diff
@@ -237,7 +237,7 @@ public void should_fail_with_parent_folder_access_after_wildcard() {
   }
 
   @Test
-  public void given_pattern_with_windows_folder_separator_listFiles_detects_wrong_pattern(){
+  public void given_pattern_with_mixed_folder_separator_and_UnixWildcard_listFiles_supports_pattern(){
```

Comment:
I think you copy pasted the UnixWildcard idea from the comment I made on `CoverageTest.java`...

if you put "mixed_folder_separator", you don't need the `UnixWildcard`, right?

Reply:
I accidentally merged two of your comments :)